lib/Doctrine/CouchDB/CouchDBClient.php

```diff
@@ -807,6 +807,33 @@ public function ensureFullCommit()
             throw HTTPException::fromResponse($path, $response);
         }
 
+        return $response->body;
+    }
+
+    /**
+     * Bulk get.
+     *
+     * @param array $docs
+     *
+     * @throws HTTPException
+     *
+     * @return array
+     */
+    public function bulkGet(array $docs = [])
```

Comment:
Can we add a type declaration here?

Reply:
I think this type of return is too complex for method documentation. We can add a link to official documentation of the method http://docs.couchdb.org/en/2.2.0/api/database/bulk-api.html#db-bulk-get

Reply:
Does it return other PHP types than `array` like the previous return annotation suggests?

Reply:
I was suggesting this
```suggestion
    public function bulkGet(array $docs = []): array
```